src/parser/flow_lexer.ml

```diff
@@ -284,6 +292,107 @@ end = struct
       with No_good -> raise e
 end
 
+module IntOfString : sig
```

Comment:
It's not 100% clear to me why this module exists

1. `int_of_string` returns a `float`. So how does this differ from `FloatOfString`?
2. There's a lot of copied code from `FloatOfString`. Can this be refactored into `FloatOfString` if we really need it?

Reply:
Yeah, removed it